hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/Deiklov/hw01_hello_otus
```

```suggestion
module github.com/Deiklov/hw_otus/hw01_hello_otus
```

Следует указать полный путь до модуля